framework/project.properties

```diff
@@ -10,7 +10,7 @@
 # Indicates whether an apk should be generated for each density.
 split.density=false
 # Project target.
-target=android-22
+target=android-23
```

Comment:
This will break http://ci.cordova.io. @alsorokin Can you please update the Windows slave to install android 23. @dblotsky Can you do the same for the iOS slave.